gpu/gpu.py

```diff
@@ -1,64 +1,108 @@
 from random import random
 
 
-def check_squared_value(x, xSquer, testValue):
-    if x != testValue - 1:
-        return False
-    elif xSquer == 1:
-        return False
-    else:
-        return True
-
+def testing_value(powerOfTwo, testValue):
+    randomValue = 2 + random() % (testValue - 4)
 
-# Run function on gpu "repetitions" cores
-def testing_value(testValue, testValueMinusOne, powerOfTwo):
-    # Choose a random number between 2 and testValue - 2
-    randomValue = int(random() * (testValue - 3)) + 2
-
-    # Calculate randomValue^testValueMinusOne mod testValue
-    x = pow(randomValue, testValueMinusOne, testValue)
+    x = pow(randomValue, powerOfTwo, testValue)
```

Comment:
powerOfTwo != testValueMinusOne

Idk co tu się dzieje teraz, ale zostawmy już szkielet, w następnych zadaniach będzie się dopieszczać to